packages/js_of_ocaml-webgpu/js_of_ocaml-webgpu.0.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Js_of_ocaml bindings for webgpu"
+description: "Js_of_ocaml bindings for webgpu"
+maintainer: ["Misha Aizatulin <mihhail.aizatulin@gmail.com>"]
+authors: ["Misha Aizatulin <mihhail.aizatulin@gmail.com>"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/tari3x/webgpu"
+doc: "https://tari3x.github.io/webgpu"
+bug-reports: "https://github.com/tari3x/webgpu"
+depends: [
+  "dune" {>= "2.2"}
+  "core" {>= "v0.12"}
+  "async" {>= "v0.12"}
+  "webidl"
+  "ppx_deriving"
+  "js_of_ocaml"
+  "js_of_ocaml-lwt"
+  "js_of_ocaml-ppx"
+  "ppx_jane"
+  "js_of_ocaml-webidl" {= version}
```

Comment:
I have also added one extra constraint: packages that are released from the same sources and depend on each other, are usually strictly requiring the same version. This is to avoid complex dependencies issues once new releases start appearing. Can you also add this to your master branch?